flipper-expressions-schema.gemspec

```diff
@@ -0,0 +1,34 @@
+# -*- encoding: utf-8 -*-
+require File.expand_path('../lib/flipper/version', __FILE__)
+require File.expand_path('../lib/flipper/metadata', __FILE__)
+
+Gem::Specification.new do |gem|
+  SCHEMAS_DIR = File.expand_path("node_modules/@flippercloud.io/expressions/schemas", __dir__)
+
+  # Ensure schemas are downloaded loaded as a git dependency
+  unless File.exist?(SCHEMAS_DIR)
+    warn "Getting schemas from @flippercloud.io/expressions..."
+    Dir.chdir(__dir__) { exec "npm install --silent" }
+  end
+
+  gem.authors       = ['John Nunemaker']
+  gem.email         = 'support@flippercloud.io'
+  gem.summary       = 'ActiveRecord adapter for Flipper'
+  gem.license       = 'MIT'
+  gem.homepage      = 'https://www.flippercloud.io/docs/expressions'
+
+  gem.files         = [
+    'package.json',
+    'package-lock.json',
+    'lib/flipper-expressions-schema.rb',
+    'lib/flipper/expression/schema.rb',
+    'lib/flipper/version.rb',
+  ] + Dir['node_modules/@flippercloud.io/expressions/schemas/*.json']
+  gem.name          = 'flipper-expressions-schema'
```

Comment:
Should this be `flipper-expression-schemas`? The ruby class is `Flipper::Expression::Schema` to avoid conflicts with functions declared in `Flipper::Expressions::*`, but maybe those should be renamed to `Flipper::Function::*`. The JS package is [`@flippercloud.io/expressions`](https://github.com/flippercloud/expressions.js)

Reply:
Are you just wondering if it should be plural? I think matching to ruby is fine even if singular.